### PR TITLE
Readme updates: 

### LINKED ISSUE #1654: Document how to proceed when a location is changing in Alma
Even though the location generation code is going to be updated in the future, it would be good to provide the steps to populate the locations in the bibdata db, when a location change happens in Alma.

### PR DESCRIPTION
**Pull request by christinach**
closes #1654 

Populate locations;
Remove deprecated db set up section;
Updated alma exlibris urls;

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,6 +155,47 @@ To run all the tests use the rake task, which sets some environment variables fo
...
+Test the updated locations in Bibdata-staging https://bibdata-staging.princeton.edu/ which is connected to the alma-sandbox; 
+The locations will not be the same as in production because they are not up to date. Deploy your branch on staging and run the following steps to make sure that nothing is breaking the tables.
+
+Connect in one of the bibdata staging boxes:
+1.`ssh deploy@bibdata-alma-staging1`
+2.`cd /opt/marc_liberation/current`
+3.`RAILS_ENV=production bundle exec rails c`
+
+Delete and repopulate the locations in the bibdata staging database:
+4.`LocationDataService.delete_existing_and_repopulate`
```
Comment (hectorcorrea) [inline README.md:186]: Should we have a rake task to do this?

File: README.md [hunk 2]
```diff
@@ -198,6 +155,47 @@ To run all the tests use the rake task, which sets some environment variables fo
...
+2.`LocationDataService.delete_existing_and_repopulate`
+
+Generate the following files: marc_to_solr/translation_maps/location_display.rb and marc_to_solr/translation_maps/locations.rb
+3.`LocationMapsGeneratorService.generate`
+
+Copy the content from the generated .rb files into the .tmpl.rb files
+4.`cp marc_to_solr/translation_maps/location_display.rb marc_to_solr/translation_maps/location_display.rb.tmpl`
+`cp marc_to_solr/translation_maps/locations.rb marc_to_solr/translation_maps/locations.rb.tmpl`
+
+Load locally the rails server and make sure that the locations have been updated.
```
Comment (hectorcorrea) [inline README.md:175]: perhaps note that we can do this by going to localhost:3000/locations/holding_locations

**Review by hectorcorrea — APPROVED** [review]
Looks good. Left a few optional notes, but looks good to me.

File: README.md (hunk 1 above)
Comment (christinach) [reply README.md:186]: I think having it in a rake task, makes it much easier for someone to run it without thinking what it's actually doing.

File: README.md (hunk 2 above)
Comment (christinach) [reply README.md:175]: yes I will add this @hectorcorrea, thank you.

File: README.md (hunk 1 above)
Comment (hectorcorrea) [reply README.md:186]: I was thinking this will make it easier to *automate* the process but I can see how this might not be desirable in this instance.

**Review by hectorcorrea — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (christinach) [reply README.md:186]: I can add as option 2 what we have in the section to update locations in a new production box. Basically to use capistrano to connect to the db.
`cap [environment] rails:console`
`LocationDataService.delete_existing_and_repopulate`

Comment (hectorcorrea) [reply README.md:186]: Good idea.

**Review by hectorcorrea — APPROVED** [review]

**Comment by christinach** [comment]
@hectorcorrea can you check again?

**Review by hectorcorrea — APPROVED** [review]

**Pull request merged**